packages/palette/src/elements/Tabs/Tabs.story.tsx

```diff
@@ -57,3 +57,32 @@ export const WithData = () => {
     </>
   )
 }
+
+export const Counts = () => {
+  return (
+    <Tabs onChange={action("onChange")}>
+      <Tab
+        name={
+          <>
+            Overview <Sup>123</Sup>
+          </>
+        }
+      />
+      <Tab
+        name={
+          <>
+            Works for sale <Sup>123</Sup>
+          </>
+        }
+      />
+      <Tab name="CV" />
+      <Tab
+        name={
+          <>
+            Auction results <Sup>123</Sup>
+          </>
+        }
+      />
+    </Tabs>
```

Comment:
This is clunky, but it looks like we're going to need these superscript counts in other places as well aside from tabs. So rather than hardcode a `count` prop, I think this is fine.